### PR TITLE
Update GLOSSARY.md

### PR DESCRIPTION
**Pull request by gconnect**
I want to contribute to the glossary.md file.

### CODE REVIEW

File: en/GLOSSARY.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ See sections [Compare things](./python_introduction/#compare-things) and [Boolea
 The front-end framework we use to [make our page pretty](./css).
 
 # class
+In object-oriented programming, a class is an extensible program-code-template for creating objects. It defines a set of properties & methods that are common to all objects of one type.
```
Comment (das-g) [inline en/GLOSSARY.md:26]: This seems more based on the general "class" concept in programming (the first sentence is probably paraphrased from [Wikipedia article "Class (computer programming)"](https://en.wikipedia.org/wiki/Class_%28computer_programming%29), isn't it?) than on [Python's specific concept of classes](https://docs.python.org/3/tutorial/classes.html).

(That's alright with me, but we should be aware of that.)

**Review by das-g — APPROVED** [review]

**Comment by gconnect** [comment]
Should the definition of the terms based on personal understanding only?
Is internet definition allowed or streamline to python definition?

**Comment by gconnect** [comment]
Don't understand this part? You’re all set—the gconnect:patch-3 branch can be safely deleted.

**Comment by das-g** [comment]
> Should the definition of the terms based on personal understanding only?

It should be based on common understanding, if possible. My question simply is whether the Python-specific definition of "class" would be more useful for readers of the Django Girls tutorial than a more general programming-language-agnostic one. But I came to the conclusion, that at least in this brevity, it doesn't really matter, so I've merged your change into my branch.

> Is internet definition allowed

Well, I guess the adopted phrasing (if it is adopted at all, and not coincidentally similar) is short enough and general enough to allow usage even without attribution, so I don't think this is a problem. But if you have any doubts about that, I can either remove the change or we could add some attribution.

> or streamline to python definition?

I don't quite understand what you mean by "streamline" here. Can you rephrase?

> Don't understand this part? You’re all set—the gconnect:patch-3 branch can be safely deleted.

This means that you can now delete your `patch-3` branch if you want, as the change introduced by it is now also on the [`glossary` branch of my repository](https://github.com/das-g/django-girls-tutorial/tree/glossary). (So the change won't be lost, even if you delete your own branch.)

**Comment by gconnect** [comment]
By streamline. I meant the definition of terms should it be limited to Python definition

**Comment by gconnect** [comment]
What happens if I don't delete the successfully merged branch?

**Comment by gconnect** [comment]
It seem like your branch is ahead of mine and the Django Girls Tutorial
repo. Because when I check glossary.md the only thing I see is code editor.
So how do I make my branch to be on the same level as yours?

**Comment by das-g** [comment]
> By streamline. I meant the definition of terms should it be limited to Python definition

I don't think we need to limit it to that. We should aim to provide the explanation most useful for participants of the workshops and other readers of the tutorial, whatever it is. Let's stick to the one you provided for now. (We can always replace it, if we can come up with a yet better one.)

> What happens if I don't delete the successfully merged branch?

Nothing. The branch just stays there in your repository.

GitHub suggests you delete it, because you shouldn't need it anymore. But if you want to keep it anyway, feel free to leave it there.
